Makefile

```diff
@@ -1,7 +1,7 @@
 GO ?= go
 GOBIN ?= $$($(GO) env GOPATH)/bin
 GOLANGCI_LINT ?= $(GOBIN)/golangci-lint
-GOLANGCI_LINT_VERSION ?= v1.63.4
+GOLANGCI_LINT_VERSION ?= v1.64.5
```

Comment:
this is not the only place that needs to be changed, rebase should give you a hint

Reply:
Bumped the one in the GitHub workflow as well